flake.nix

```diff
@@ -31,13 +31,9 @@
         commitHash = toString (self.rev or self.dirtyRev);
         npmDepsHash = builtins.readFile ./npmDepsHash;
 
-        polykey-cli-app = pkgs.buildNpmPackage {
+        polykey-cli = pkgs.buildNpmPackage {
           inherit npmDepsHash;
           pname = utils.packageName;
-          meta = {
-            # Other meta attributes like description, homepage, license, etc.
-            mainProgram = "polykey";
-          };
```

Comment:
Turns out this was not needed because of the `type: "app"` in the `apps`. It then just uses the `program` attribute. This is only needed if we just refer to a derivation directly

Reply:
Okay then.